GNUmakefile

```diff
@@ -1,5 +1,7 @@
 VERSION ?= 0.3.0
 PREFIX ?= /usr/local
+INSTALL ?= $(shell command -v ginstall 2>/dev/null || echo install)
+CP ?= $(shell command -v gcp 2>/dev/null || echo cp)
```

Comment:
No, this is not needed.  `cp -P` is POSIX.  `cp -r` isn't because this can easily be converted to `cp -R`.

Reply:
This is not true...

Reply:
https://pubs.opengroup.org/onlinepubs/9699919799/utilities/cp.html

Reply:
So?

With gcp:
```
.rwxr-xr-x  112k tobias  7 Feb 16:53  libbemenu.so.0.3.0
lrwxr-xr-x    18 tobias  7 Feb 16:53  libbemenu.so -> libbemenu.so.0.3.0
lrwxr-xr-x    18 tobias  7 Feb 16:53  libbemenu.so.0 -> libbemenu.so.0.3.0
```

With FreeBSD's cp:
```
.rwxr-xr-x  112k tobias  7 Feb 16:54  libbemenu.so
.rwxr-xr-x  112k tobias  7 Feb 16:54  libbemenu.so.0
.rwxr-xr-x  112k tobias  7 Feb 16:54  libbemenu.so.0.3.0
```

Notice how the library is installed three times without any symlink in sight?

Reply:
It seems FreeBSD would need to use `cp -RP` not just `cp -P`.

> If the -R option was not specified, cp shall take actions based on the type and contents of the file referenced by the symbolic link, and not by the symbolic link itself, **unless the -P option was specified**.

From FreeBSD's manual it seems to have concluded that `-P` shall only have an effect if `-R` is also provided.  That's just broken and FreeBSD ought to be obliged to fix this. It is, in my opinion, unreasonable to expect others to turn their projects into `ifdef` hellscapes to satisfy platform idiosyncrasies.

Reply:
`cp -RP` might be worth a shot.